src/main/java/org/cbio/gdcpipeline/JobConfiguration/BatchConfiguration.java

```diff
@@ -0,0 +1,90 @@
+package org.cbio.gdcpipeline.JobConfiguration;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.cbio.gdcpipeline.tasklet.FileMappingTasklet;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.core.listener.ExecutionContextPromotionListener;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.Resource;
+
+@Configuration
+public class BatchConfiguration {
+
+    private static Log LOG = LogFactory.getLog(BatchConfiguration.class);
+
+    @Autowired
+    JobBuilderFactory jobBuilderFactory;
+
+    @Autowired
+    StepBuilderFactory stepBuilderFactory;
+
+    @Resource(name = "runClinicalDataStep")
+    Step runClinicalDataStep;
+
+    @Resource(name = "runClinicalMetaDataStep")
+    Step runClinicalMetaDataStep;
+
+
+    @Value("${chunk.interval}")
+    private int chunkInterval;
+
+    @Bean
+    public ExecutionContextPromotionListener jobExecutionListener() {
```

Comment:
I've tried doing that but doesn't work. This seems to be the recommended way to do it. [Link](https://stackoverflow.com/questions/2292667/how-can-we-share-data-between-the-different-steps-of-a-job-in-spring-batch). 
 Although, I will add a TODO to research on data size limitations. I read some comments raising concern over data size that it shouldn't be large.